pkg/data/run_store.go

```diff
@@ -233,6 +239,28 @@ func (s *storer) scenarioData(scenarioRunId int64) error {
 		}
 	}
 
+	cpuUtilizationStmt, err := s.conn.Prepare(`insert into cpu_utilizations(
+		cpu_utilization
+	  , calculated_at
```

Comment:
There's a tabs vs spaces mismatch here.

Reply:
Set up my IDE to avoid this things. See e91267565b8ba92593c89fe969f1d65a35e99baf commit.